pages/index.html

```diff
@@ -0,0 +1,138 @@
+{% extends "base.html" %}
+
+{% block page_title %}Rise25 - Meet the 2024 honorees{% endblock %}
+{% block page_desc %}{% endblock %}
+
+{% block page_css %}
+<link rel="stylesheet" href="/media/css/index.scss">
+{% endblock %}
+
+{% macro category(id, theme_class, title, desc, expanded, link) %}
+<div id="{{ id }}" class="r25-c-category {{ theme_class }}">
+  <div class="r25-c-category-body">
+    <h3 class="r25-c-category-title">{{ title }}</h3>
+    <p class="r25-c-category-desc">{{ desc }}</p>
+    <p class="r25-c-category-expand">
+      {{ expanded }}
+    </p>
+  </div>
+
+  <p class="r25-c-category-cta">
+    <a class="r25-c-button t-dark" href="{{ link }}">See the honorees</a>
+  </p>
+</div>
+{% endmacro %}
+
+{% block content %}
+<section class="r25-hero">
+  <div class="r25-background-overlay"></div>
+  <video class="r25-background-video" preload="auto" playsinline autoplay loop muted width="680" height="383">
+    <source src="https://assets.mozilla.net/rise25/rise25-2024-hero-loop.webm" type="video/webm">
+  </video>
+
+  <img class="r25-background-video-fallback" src="/media/img/hero-fallback.jpg" width="1600" height="900" alt="">
+
+  <div class="mzp-l-content">
+    <div class="r25-hero-content">
+      <h1 class="r25-c-wordmark">Rise25</h1>
+      <h2 class="r25-fancy-title">Meet the Honorees</h2>
```

Comment:
**Suggestion (a11y):** On smaller screens, some letters on the title doesn't look as clear with the background linear gradient. Would it be possible to slightly darken the gradient, so the letters are more distinguishable? It's just subtle, but good to cover ground if we can

<kbd>![Screenshot 2024-06-24 at 3 33 07 PM](https://github.com/mozilla/rise25/assets/42309026/0d5e0d20-789c-46c9-b831-103c1adf364d)</kbd>